.buildkite/pipelines/on_merge.yml

```diff
@@ -254,7 +254,7 @@ steps:
   - command: .buildkite/scripts/steps/check_types.sh
     label: 'Check Types'
     agents:
-      queue: n2-16-spot
+      queue: n2-2-spot
```

Comment:
https://buildkite.com/elastic/kibana-pull-request/builds/171354#018b6d91-3a82-4b9e-b85f-c53b23c98042

The larger machine scales node up to more memory, but we're not making use of most of the CPU.